css/civicrmNavigation.css

```diff
@@ -91,6 +91,7 @@ li.menu-separator.active{
   display:block;
   font-weight:normal;
   text-decoration:none;
+  border-bottom-style:none;
```

Comment:
bascially i found that the default theme i was using after doing a civibuild with d8 and hacking about to get the menu working was adding the bartik theme out of the box adds ... onto a.link so this corrects it